R/score_forecasts.R

```diff
@@ -0,0 +1,110 @@
+#' Score forecasts
+#'
+#' @param forecasts required data.frame with forecasts in the format returned
+#' by load_forecasts
```

Comment:
in the documentation function names should be documented using `\code{\link{functioname}}` or `\code{\link[packagename]{functioname}}.

for more info see: https://r-pkgs.org/man.html#roxygen-comments